### PR TITLE
net-mail/qmail-notify: revbump with EAPI 7

### PR DESCRIPTION
**Pull request by arkamar**
I would like to proxy-maint this package. Should I remove an old revision?

### CODE REVIEW

File: net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	${DEPEND}
+	virtual/cron
+	virtual/qmail
+"
+
+src_prepare() {
+	eapply_user
+
+	echo "$(tc-getCC) ${CFLAGS}" > conf-cc
+	echo "$(tc-getCC) ${LDFLAGS}" > conf-ld
```
Comment (DerDakon) [inline net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild]: this probably also needs ar, see qmail eclass.

File: net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+HOMEPAGE="http://untroubled.org/qmail-notify/"
+
+SLOT="0"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~hppa ~ppc ~sparc ~x86"
+IUSE=""
+
+DEPEND=""
+RDEPEND="
+	${DEPEND}
```
Comment (DerDakon) [inline net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild]: merge this two lines

File: net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+src_install () {
+	exeinto /usr/sbin
+	doexe qmail-notify
+
+	exeinto /etc/cron.hourly
+	doexe "${FILESDIR}"/qmail-notify.cron
+
+	dodoc README ANNOUNCEMENT TODO cron.hourly NEWS VERSION
```
Comment (DerDakon) [inline net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild]: I have not looked into these files, but VERSION is probably meaningless. Also look if TODO is really worth installing, at least for qmail it really isn't.

File: net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild (hunk 1 above)
Comment (arkamar) [reply net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild]: Yes, you are right, it needs ar and ranlib as well.

File: net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild (hunk 2 above)
Comment (arkamar) [reply net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild]: ok

File: net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild (hunk 3 above)
Comment (arkamar) [reply net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild]: VERSION and TODO are not worth of installing. How about cron.hourly? It is basically script with the same functionality of the one we install from `${FILESDIR}/qmail-notify.cron`.

Comment (DerDakon) [reply net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild]: Your choice as maintainer ;) I just tell you what I have found when renovating netqmail.

Comment (arkamar) [reply net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild]: Ok, thanks!

File: net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+	virtual/cron
+	virtual/qmail
+"
+
+src_prepare() {
+	eapply_user
+
+	echo "$(tc-getCC) ${CFLAGS}" > conf-cc
+	echo "$(tc-getCC) ${LDFLAGS}" > conf-ld
+	sed -e "#'ar #'$(tc-getAR) #" -e "s#'ranlib #'$(tc-getRANLIB) #" -i Makefile
```
Comment (juippis) [inline net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild]: This needs to be appended with `|| die` in case it fails to run.

Comment (arkamar) [reply net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild]: Thanks, it is fixed.

**Comment by juippis** [comment]
Thanks for taking care of it!

Thanks @DerDakon for reviewing. I'll remove the package.mask entry.

**Pull request closed without merging**